cns/restserver/restserver.go

```diff
@@ -19,6 +21,7 @@ import (
 	"github.com/Azure/azure-container-networking/cns/types/bounded"
 	"github.com/Azure/azure-container-networking/cns/wireserver"
 	acn "github.com/Azure/azure-container-networking/common"
+	"github.com/Azure/azure-container-networking/crd/nodenetworkconfig/api/v1alpha"
```

Comment:
I don't love that this is being imported in to the `restserver` package. We have made it this far without having the restserver aware of the CRDs at all, let's try to keep it that way.

I think this import is only for the status codes? They should probably be defined in a higher level package (or with all our existing common types in the [CNS package](https://github.com/Azure/azure-container-networking/blob/master/cns/NetworkContainerContract.go)), then they can be used in both the restserver and the CRD packages.